data_storage/export_format.md

```diff
@@ -1,14 +1,16 @@
 Export Format
 -------------
 
-- ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- ![](/badge/req.png) A Toxcore data file saved in the directory selected by
+  the user, with the file name `<profile_name>.tox`
```

Comment:
UX is not the concern of the standard. The focus is on client compatibility and security, as the introduction says.